js/gateio.js

```diff
@@ -2778,7 +2807,7 @@ module.exports = class gateio extends Exchange {
                         'side': side,
                         'price': this.priceToPrecision (symbol, price),
                         'amount': this.amountToPrecision (symbol, amount),
-                        'account': account, // normal, margin
+                        'account': marginType,
```

Comment:
I think, there's a bit of a confusion here, we probably should leave that line intact, cause it's for conditional orders which are not tested in your verbose output, and the values here are "normal" and "margin". The rules for the `account` values are different between regular orders and conditional orders. It would be great if we could test all families (conditional cross margin, conditional isolated margin, conditional spot, regular cross margin, regular isolated margin, regular spot).

- https://www.gate.io/docs/apiv4/en/#create-a-price-triggered-order

<img width="986" alt="Screenshot 2022-05-03 at 14 18 59" src="https://user-images.githubusercontent.com/1294454/166444284-7e03941d-f068-4779-99be-56931d8a3307.png">

Reply:
https://github.com/ccxt/ccxt/pull/13083/files#r864294160

Reply:
> (conditional cross margin, conditional isolated margin, conditional spot, regular cross margin, regular isolated margin, regular spot).

I'm posted the output for all 6 in the original comment